cspell.config.json

```diff
@@ -62,6 +62,8 @@
     "errgroup",
     "ectx",
     "reqs",
-    "struct"
+    "struct",
+    "Pagainating",
+    "pagainating"
```

Comment:
Could you actually remove these from the cspell exclusions, the correct spelling is `Paginating` without the extra "a" after the "g"

Reply:
oh my bad, I didn't know how to spell
Lemme fix that 👍

Reply:
All fixed 👍